Makefile.am

```diff
@@ -512,6 +519,11 @@ if ENABLE_BACKEND_KINESIS
 else
     netdata_LINK = $(CCLD) $(CFLAGS) $(LDFLAGS) -o $@
 endif
+if ENABLE_BACKEND_PROMETHEUS_REMOTE_WRITE
+    netdata_LINK = $(CXXLD) $(CXXFLAGS) $(LDFLAGS) -o $@
+else
+    netdata_LINK = $(CCLD) $(CFLAGS) $(LDFLAGS) -o $@
```

Comment:
all netdata_LINK should be part of a single if-else tree, but have you confirmed that is necessary to override the linker for this?

Reply:
When I was working on Kinesis backend, I couldn't link the daemon without the hack with `_LINK`. Though,  of course, it would be better to get rid of this override.